lib/mix/tasks/install.ex

```diff
@@ -0,0 +1,175 @@
+  @moduledoc "Installer Mix task for LiveView Native: `mix lvn.install`"
+  use Mix.Task
+
+  @requirements ["app.config"]
+
+  @shortdoc "Installs LiveView Native for SwiftUI."
+  def run(args) do
+    {parsed_args, _, _} = OptionParser.parse(args, strict: [host_project_config: :string, task_settings: :string])
+    host_project_config = Keyword.fetch!(parsed_args, :host_project_config)
+    task_settings = Keyword.fetch!(parsed_args, :task_settings)
+
+    make_native_project_dir(host_project_config)
+    copy_xcodegen_files(host_project_config)
+    replace_app_name_in_sources(host_project_config)
+    rename_sources_directory(host_project_config)
+    run_xcodegen(host_project_config, task_settings)
+    remove_xcodegen_files(host_project_config)
+
+    :ok
+  end
+
+  def lvn_install_config, do: %{
+    client_name: "SwiftUI",
+    mix_config: %{
+      live_view_native: [
+        plugins: [LiveViewNative.SwiftUI]
+      ],
+      live_view_native_stylesheet: [
+        parsers: [
+          swiftui: LiveViewNative.SwiftUI.RulesParser
+        ]
+      ]
+    },
+    prompts: [
+      install_xcodegen: %{
+        type: :confirm,
+        label: "Xcodegen is required to generate an Xcode project for your app. Would you like to install it?",
+        ignore: System.find_executable("xcodegen") != nil,
+        on_yes: &install_xcodegen/0,
+        on_no: &skip_swiftui_install/0
+      },
+      targets: %{
+        type: :multiselect,
+        label: "Select any number of deployment targets for your app.",
+        options: ["iOS", "macOS", "watchOS"],
+        default: ["iOS"],
+        default_label: "1"
+      }
+    ]
+  }
+
+  ###
+
+  defp install_xcodegen do
+    cond do
+      # Install with Homebrew
+      System.find_executable("brew") ->
+        status_message("running", "brew install xcodegen")
+        System.cmd("brew", ["install", "xcodegen"])
+        true
+
+      # Install with Mint
+      System.find_executable("mint") ->
+        status_message("running", "mint install yonaskolb/xcodegen")
+        System.cmd("mint", ["install", "yonaskolb/xcodegen"])
+        true
+
+      # Clone from GitHub (fallback)
+      true ->
+        File.mkdir_p("_build/tmp/xcodegen")
+        status_message("running", "git clone https://github.com/yonaskolb/XcodeGen.git")
+        System.cmd("git", ["clone", "https://github.com/yonaskolb/XcodeGen.git", "_build/tmp/xcodegen"])
+        true
+    end
+  end
+
+  defp make_native_project_dir(%{native_path: native_path}) do
+    native_path
+    |> Path.join("swiftui")
+    |> File.mkdir_p()
+  end
+
+  defp copy_xcodegen_files(%{native_path: native_path}) do
+    priv_dir = :code.priv_dir(:live_view_native_swift_ui)
+    native_project_dir = Path.join(native_path, "swiftui")
+    xcodegen_path = Path.join(priv_dir, "xcodegen")
+
+    File.cp_r(xcodegen_path, native_project_dir)
+  end
+
+  defp replace_app_name_in_sources(%{app_namespace: app_namespace, native_path: native_path}) do
```

Comment:
Can something similar be done for the `example.com` URL?

Reply:
Oops forgot about this. Most definitely.